routes/web.php

```diff
@@ -11,6 +11,45 @@
 |
 */
 
+// @todo move to helper
+if (!function_exists('resource')) {
+    function resource($router, $uri, $controller, array $options = [])
```

Comment:
@willdoran this is modelled off laravels `Route::resource()` function. It needs to move elsewhere but I wanted to validate the idea first